server/routes.go

```diff
@@ -410,6 +415,11 @@ func EmbeddingHandler(c *gin.Context) {
 		return
 	}
 
+	if model.IsEmbedding() {
```

Comment:
This should be in the `/chat` endpoint. It blocks generating embeddings with embedding models.

Reply:
Thanks! Wrong handler...